orca-kayenta/src/main/java/com/netflix/spinnaker/orca/kayenta/pipeline/functions/ConfigExpressionFunctionProvider.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.kayenta.pipeline.functions;
+
+import com.google.common.base.Strings;
+import com.netflix.spinnaker.kork.api.expressions.ExpressionFunctionProvider;
+import com.netflix.spinnaker.kork.expressions.SpelHelperFunctionException;
+import com.netflix.spinnaker.orca.kayenta.KayentaCanaryConfig;
+import com.netflix.spinnaker.orca.kayenta.KayentaService;
+import java.util.List;
+import java.util.Optional;
+import javax.annotation.Nullable;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ConfigExpressionFunctionProvider implements ExpressionFunctionProvider {
+
+  // Static because it's needed during expression eval (which is a static)
+  private static KayentaService kayentaService;
+
+  public ConfigExpressionFunctionProvider(KayentaService kayentaService) {
+    this.kayentaService = kayentaService;
+  }
+
+  @Nullable
+  @Override
+  public String getNamespace() {
+    return null;
+  }
+
+  @NotNull
+  @Override
+  public Functions getFunctions() {
+    return new Functions(
+        new FunctionDefinition(
+            "acaNameToId",
+            "Look up the ACA config ID for the given Name and App",
```

Comment:
"ACA" is not an acronym that OSS is familiar with. Can we choose something else (would `canaryNameToId` make sense?)

Reply:
Sure, can rename to canaryConfigNameToId -- thanks!

Reply:
Done!